addon/adapters/drf.js

```diff
@@ -111,5 +96,21 @@ export default DS.RESTAdapter.extend({
     } else {
       return error;
     }
+  },
+
+  /**
+   * This is used by RESTAdapter.groupRecordsForFindMany.
+   *
+   * The original implementation is a complex stripping of the id from
+   * the URL, which can be dramatically simplified by just returning
+   * the base URL for the type.
+   *
+   * @method _stripIDFromURL
+   * @param {DS.Store} store
+   * @param {DS.Model} record
+   * @return {String} url
+   */
+  _stripIDFromURL: function(store, record) {
+    return this.buildURL(record.constructor.typeKey);
   }
```

Comment:
Yes, I originally started working on the bug reported in Ember Data, but after digging into it I decided that Ember Data's RESTAdapter really shouldn't be modified to support trailing slashes in this method, since it does not support them anywhere else.  Doing so should be left to third-party adapters (like ours).

Furthermore, after examining the RESTAdapter implementation, I've concluded that they are horribly overcomplicating it.  I'm not sure if this is a result of various code changes over time, or if I'm missing something really obvious.  My intention is to comment on the ember-data issue once this PR is merged (so I can reference it).